src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/TripTestsIT.java

```diff
@@ -0,0 +1,138 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.gtfs;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Response;
+
+import fi.otavanopisto.kuntaapi.server.integrations.gtfs.GtfsConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.PtvConsts;
+import fi.otavanopisto.kuntaapi.test.AbstractIntegrationTest;
+
+public class TripTestsIT extends AbstractIntegrationTest {
+  
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    createPtvSettings();
+    
+    getPtvMocker()
+      .mockOrganizations("0de268cf-1ea1-4719-8a6e-1150933b6b9e")
+      .startMock();
+
+    waitApiListCount("/organizations", 1);
+    
+    createGtfsSettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/transportTrips", getOrganizationId(0)), 7); 
+  }
+
+  @After
+  public void afterClass() {
+    String organizationId = getOrganizationId(0);
+    getPtvMocker().endMock();
+    deletePtvSettings();
+    deleteGtfsSettings(organizationId);
+  }
+
+  @Test
+  public void testListTrips() {
+    String organizationId = getOrganizationId(0);
+    
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportTrips", organizationId);
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(7))
+      .body("id[1]", notNullValue())
+      .body("routeId[1]", notNullValue())
+      .body("scheduleId[1]", notNullValue())
+      .body("headsign[1]", is("Mikkeli matkakeskus"));
+    
+    String routeId = response.body().jsonPath().getString("routeId[1]");
+    String ScheduleId = response.body().jsonPath().getString("scheduleId[1]");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)